config/default.json

```diff
@@ -19,7 +19,9 @@
       { "period": "5 minutes", "limit": 0 }
     ]
   },
-  "message_id_hmac_key": "YOU MUST CHANGE ME",
+  "messagedata": {
+      "hmackey": "YOU MUST CHANGE ME"
```

Comment:
Fwiw, I maintain my position that we don't need nesting for this and could just have it at the top level (see discussion in https://github.com/mozilla/fxa-email-service/pull/72#discussion_r196182241). Not sure what everyone else thinks though, so ignore me if you guys disagree.

Reply:
If we really want to keep the nesting though, maybe it should be inverted in this case. Make `hmackeys` the parent property and then we can have all the different instances that we imagine we want underneath it as children.

Reply:
I agree that the nesting is a bit much, but I still think `FXA_EMAIL_MESSAGEDATAHMACKEY` is just too long. I do like the initial solution that you had for this, `FXA_EMAIL_HMACKEY`. I will take a look at the environment separator you mentioned and see if we can keep the underscores after all, if we don't, I will still remove the nesting :)

Reply:
Oh yeah, I definitely didn't mean `FXA_EMAIL_MESSAGEDATAHMACKEY` that's way too ugly. But `FXA_EMAIL_HMACKEY` is fine I think.

Reply:
(but I think @vladikoff disagrees, so let's see what he says)

Reply:
I'm sold! I just want my ENV vat :)